package.json

```diff
@@ -1,16 +1,46 @@
 {
   "private": true,
-  "workspaces": [
-    "dataregistry",
-    "dataregistry-extension"
-  ],
+  "workspaces": {
+    "packages": [
+      "packages/*"
+    ]
+  },
   "scripts": {
-    "build": "tsc --build dataregistry dataregistry-extension && jlpm workspace @jupyterlab/dataregistry-extension run generateSchema",
-    "build:dev": "jlpm install && jlpm clean && jlpm run build && jlpm run link",
-    "lint":  "jlpm workspace @jupyterlab/dataregistry-extension lint",
-    "clean": "jlpm workspace @jupyterlab/dataregistry run clean && jlpm workspace @jupyterlab/dataregistry-extension run clean",
-    "build:watch": "tsc --build --watch  dataregistry dataregistry-extension",
-    "link": "jupyter labextension link dataregistry --no-build && jupyter labextension link dataregistry-extension",
-    "unlink": "jupyter labextension unlink dataregistry --no-build && jupyter labextension unlink dataregistry-extension"
+    "all": "jlpm run clean && jlpm run build && jlpm run build:jupyter",
+    "all:watch": "jlpm run clean && jlpm run build && jlpm run build:jupyter:watch",
+    "build": "jlpm run build:dev",
+    "build:dev": "jlpm install && jlpm run clean:packages && lerna link && jlpm run build:packages && jlpm run link:packages",
+    "build:jupyter": "YARN_REGISTRY=\"http://localhost:4873\" jupyter lab build",
+    "build:jupyter:watch": "YARN_REGISTRY=\"http://localhost:4873\" jupyter lab --watch",
+    "build:packages": "lerna run build",
+    "build:watch": "tsc --build --watch --listEmittedFiles packages/*",
+    "clean": "jlpm run clean:jupyter && jlpm clean:packages && jlpm clean:node",
+    "clean:jupyter": "jlpm run uninstall:extensions && (lerna run --no-bail unlink || echo 'At least one unlink command failed, but continuing...')",
+    "clean:node": "lerna clean --yes && rimraf node_modules",
+    "clean:packages": "lerna run clean",
+    "link:packages": "lerna run link",
+    "lint": "jlpm run lint:css && jlpm run lint:typescript",
+    "lint:css": "stylelint packages/**/*.css",
+    "lint:typescript": "eslint packages/**/src/*.ts",
+    "rebuild:packages": "jlpm run clean:packages && jlpm run build:packages",
+    "registry": "verdaccio",
+    "registry:init": "npm adduser --registry http://localhost:4873 && lerna exec --scope dataregistry-csvviewer --scope dataregistry-registry -- npm publish --registry http://localhost:4873",
+    "uninstall:extensions": "jupyter labextension uninstall --all --no-build",
+    "unlink:packages": "lerna run unlink"
+  },
+  "devDependencies": {
```

Comment:
Do you have a sense of what dev dependencies should be in the root package vs the child packages? It seems like some are added in both places.

Reply:
Yeah, I can remove `rimraf` and `typescript` from the individual packages, I believe.